modules/core/src/main/java/org/apache/ignite/internal/commandline/ClusterStateChangeCommand.java

```diff
@@ -91,6 +98,18 @@
         catch (IllegalArgumentException e) {
             throw new IllegalArgumentException("Can't parse new cluster state. State: " + s, e);
         }
+
+        force = false;
```

Comment:
Do we need this line? Default `force` value is `false` already.

Reply:
> Do we need this line? Default `force` value is `false` already.

Yes, we need this. Commands are statefull unfortunately. Without this, next command usage will go with previous value 'force' param.